wrappers/cppopencv/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.8.2)
+
+set(THREADS_USE_PTHREADS_WIN32 true)
+find_package(Threads)
+find_package( OpenCV REQUIRED )
+
+include_directories(. ${THREADS_PTHREADS_INCLUDE_DIR})
```

Comment:
Is that needed?

Reply:
It should not be necessary - I think `THREADS_PTHREADS_INCLUDE_DIR` does not even exist in CMake anymore.  But since you already require cxx_std_11, you could use `std::thread` and `std::mutex` instead of pthread in the code.  Then there would be no need to find Threads package at all.